### ISSUE TITLE
As a speaker, I want to be able to add 'other' talks to my talk list, so that I can keep a list of all talks I've delivered in one place

### ISSUE THREAD

**Issue by liam-wiltshire** [joindin/joindin-web2#823]
Something that I'd thought about previously, but never had the time to do.

There are events (conferences, user groups etc) that don't used joind.in. As a speaker, I'd like to have one place where all my talks are listed, as at the moment I have some on joind.in, some in other lists etc. Even more so if I was a new speaker, being able to share a link to my joind.in profile that shows I've spoken at usergroups or small events that do not list on joind.in would potentially be useful.

I would suggest that 'standalone' talks should not be able to have reviews, as it would make it potentially easier to 'game' the system, and more just functions as a 'historical record'.

Not sure it's something that needs to be a priority now, but if anyone sees value in it then happy to look at putting a PR together for it.

**Comment by heiglandreas** [comment]
Why not have comments on them when the talk has just been given? That way a speaker could use joind.in for feedback even though the event itself is not using it...

**Comment by liam-wiltshire** [comment]
@heiglandreas my concern with that is that there is no 'oversight' - so someone could create a load of talks, then add fake 5 star reviews to them to perhaps push poorly received talks down the list or whatever - perhaps I'm being over-cautious about it. With the normal process, the conference owner has to add the talk, then approve the claim, so there is that check-and-balance process going on.

I did originally think, like you, that it could be open for comments. I guess it's a judgement call to consider is that more helpful than the risk of it being abused.

**Comment by heiglandreas** [comment]
As always. One cod f. e. allow feedback for people that have x "regular" talks. Though on the other hand, those probably do not need that... 🤔

**Comment by liam-wiltshire** [comment]
Yeah, exactly. In my specific use-case, I'm not worried about not having feedback on those talks, I'd just like to list my ConFoo, user-group etc talks in joind.in, so I can use that as my 'source of truth' in future if I ever need to. I guess for others, being able to get feedback on those talks is useful.

Perhaps if that's the case, we just mark them with a 'self listed' indicator of some description, so if a speaker has 5 'proper' conference talks with average scores of 2-3, and 5 'self listed' talks with straight fives, a conference organiser at least has the full picture to hand?
